src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -90,14 +90,15 @@ public GroupDialogView(DialogService dialogService, BibDatabaseContext currentDa
 
     @FXML
     public void initialize() {
+        System.out.println("Group Dialog View initiated");
         hierarchyText.put(GroupHierarchyType.INCLUDING, Localization.lang("Union"));
         hierarchyToolTip.put(GroupHierarchyType.INCLUDING, Localization.lang("Include subgroups: When selected, view entries contained in this group or its subgroups"));
         hierarchyText.put(GroupHierarchyType.REFINING, Localization.lang("Intersection"));
         hierarchyToolTip.put(GroupHierarchyType.REFINING, Localization.lang("Refine supergroup: When selected, view entries contained in both this group and its supergroup"));
         hierarchyText.put(GroupHierarchyType.INDEPENDENT, Localization.lang("Independent"));
         hierarchyToolTip.put(GroupHierarchyType.INDEPENDENT, Localization.lang("Independent group: When selected, view only this group's entries"));
 
-        nameField.textProperty().bindBidirectional(viewModel.nameProperty());
+      //  nameField.textProperty().bindBidirectional(viewModel.nameProperty());
```

Comment:
Disabling the binding of this field to the ViewModel would make the user unable to change the name of a group. Please restore the binding.